modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -378,19 +348,24 @@ The Invoke APEX Rest operation enables users to invoke a method from an Apex cla
 ----
 
-The `body` element contains `URLParameters`, which is a map containing the parameters that replace the wildcards in the path of the REST resource described in the Apex class. For example if the REST resource is set to `@RestResource(urlMapping='/myResource/\*/mySubResource/*')`, then the value of `Parameter1` replaces the first `\*` and the value of `Parameter2` replaces the second `*`. The key names must start with `Parameter` followed by a number showing the position of the `*` to be replaced. After the `URLParameters` block, provide the content of the `body` to be sent to the REST resource, as shown in the example `account` block.
-
-The `headers` and `cookies` fields describe the headers and cookies to pass along with the HTTP request to the desired service.
+In this example:
 
-The `queryParameters` field describes the query parameters to use, and the keys and values in this map that have to be accepted by the specified Apex Class.
+* The `body` element contains `URLParameters`, which is a map containing the parameters that replace the wildcards in the path of the REST resource described in the Apex class.
++
+For example if the REST resource is set to `@RestResource(urlMapping='/myResource/\*/mySubResource/*')`, the value of `Parameter1` replaces the first `\*`, and the value of `Parameter2` replaces the second `*`.
+* Key names must start with `Parameter`, followed by a number that shows the position of the `*` to be replaced.
++
+* After the `URLParameters` block, provide the content of the `body` to send to the REST resource, as shown in the example `account` block.
+* The `headers` and `cookies` fields describe the headers and cookies to pass along with the HTTP request to the desired service.
+* The `queryParameters` field describes the query parameters to use, and the keys and values in this map that the specified Apex Class must accept.
 
 === Insert Values in a Salesforce Drop-Down Menu
 
-Inserting dependent values into an existing drop-down menu in Salesforce does not always work. Test to confirm functionality.
+Inserting dependent values into an existing drop-down menu in Salesforce does not always work. Test to confirm this functionality.
 
 === Evaluate Values in a Salesforce Drop-Down Menu
 
-If you are evaluating against a value in an existing drop-down list field in Salesforce, be sure to use the exact value in the drop-down list. For example, if you use the value `US` to evaluate against the contents of a drop-down list that contains the value `USA`, the evaluation works, but the result is two values in the drop-down list: one for `US` and one for `USA`.
+If you are evaluating against a value in an existing drop-down list field in Salesforce, use the exact value in the drop-down list. For example, if you use the value `US` to evaluate against the contents of a drop-down list that contains the value `USA`, the evaluation works, but the result is two values in the drop-down list: one for `US` and one for `USA`.
```

Comment:
GLOBALLY evaluate whether the audience requires us to specify "drop-down" for lists and menus. Even I understand what the little downward-facing arrow means!  lol

Reply:
In addition to the above recommendation (not reflected here)...:
```suggestion
If you are evaluating against a value in an existing drop-down list field in Salesforce, use the exact value in the list. For example, if you use the value `US` to evaluate against the contents of a drop-down list that contains the value `USA`, the evaluation works, but the result is two values: one for `US` and one for `USA`.
```